### ISSUE TITLE
[Test Failure] test_batch_whitened_svgp_gp_regression

### ISSUE THREAD

**Issue by Balandat** [cornellius-gp/gpytorch#483]
Getting the following test failure on master: 

```
AssertionError: 0.14334218204021454 not less than 0.1
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/data/users/balandat/fbsource/fbcode/buck-out/dev/gen/pytorch/gpytorch/test_gpytorch_examples#binary,link-tree/test/examples/test_batch_whitened_svgp_gp_regression.py", line 167, in test_regression_error_cuda
    self.assertLess(mean_abs_error2.item(), 1e-1)
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 1208, in assertLess
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
```

**Comment by jacobrgardner** [comment]
@Balandat I'm unable to reproduce this on today's pytorch-nightly build and gpytorch master. What environment are you using? I'm

**Comment by Balandat** [comment]
@jacobrgardner This is on gpytorch master and the latest bleeding edge pytorch(well, 20 mins old or so). Let me see if fixing the seed fixes the test and still works for github contbuild

**Comment by KeAWang** [comment]
I'm also failing this test in #484. I'm on gpytorch master branch with Python 3.6.7 and a pretty old version of pytorch nightly
```
>>> torch.__version__
'1.0.0.dev20181220'
```
The unittest error:
```
Traceback (most recent call last):
  File ".../gpytorch/test/examples/test_batch_whitened_svgp_gp_regression.py", line 167, in test_regression_error_cuda
    self.assertLess(mean_abs_error2.item(), 1e-1)
AssertionError: 0.1433437168598175 not less than 0.1
```

**Comment by KeAWang** [comment]
Somehow running the unittest locally fails but the Travis CI passes?

**Comment by gpleiss** [comment]
Travis doesn't run the CUDA tests (because it doesn't have GPUs). The test that's failing is on CUDA.
It is strange that the CPU version of the test doesn't fail.

**Comment by Balandat** [comment]
Yeah so this happens also when changing the CUDA seed - so it doesn't just seem to be some changes related to that.

**Comment by KeAWang** [comment]
This also fails on pytorch stable 1.0.0 with CUDA

**Comment by KeAWang** [comment]
closed by #493
